### DISCUSSION TITLE
Develop a crystal lattice from the unit cell information

### DISCUSSION THREAD

**Discussion by roshan2004** [MDAnalysis/mdanalysis#4579] [General]
Hi everyone, can MDAnalysis read a cif file, and allow to develop a super cell and eventually a crystal lattice from the unit cell information provided in the corresponding cif file? Thanks

**Comment by hmacdope** [comment]
@roshan2004 this is theoretically possible but difficult in MDAnalysis. I would reccomend the Atomic Simulation Environment (ASE: https://wiki.fysik.dtu.dk/ase/) for this purpose, I have done something similar using ASE before.
